config/display.ini

```diff
@@ -94,3 +94,44 @@ lib_deps_external =
 lib_ignore_builtin =
     HalLedMatrix
 lib_ignore_external =
+
+
+; ********************************************************************************
+; M5Stack 
+; https://shop.m5stack.com/collections/m5-controllers/CORE
+; Configuration based on https://docs.m5stack.com/en/core/gray
+; ********************************************************************************
+[display:m5stack_core]
+build_flags =
+    -D USER_SETUP_LOADED=1
+    -D ILI9341_DRIVER
+    -D M5STACK
+    -D TFT_WIDTH=320
+    -D TFT_HEIGHT=240
+    -D CGRAM_OFFSET=1
+    -D TFT_MISO=19
+    -D TFT_MOSI=23
+    -D TFT_SCLK=18
+    -D TFT_CS=14
+    -D TFT_DC=27
+    -D TFT_RST=33
+    -D TFT_BL=32
+    -D TFT_BACKLIGHT_ON=HIGH
+    -D SPI_FREQUENCY=27000000
+    -D SPI_READ_FREQUENCY=5000000
+    -D TFT_INVERSION_ON
+    -D DISABLE_ALL_LIBRARY_WARNINGS
+    -D TFT_MATRIX_WIDTH=32U
+    -D TFT_MATRIX_HEIGHT=8U
+    -D TFT_PIXEL_WIDTH=6
+    -D TFT_PIXEL_HEIGHT=6
+    -D TFT_PIXEL_DISTANCE=1
+    -D TFT_DEFAULT_BRIGHTNESS=100
+lib_deps_builtin =
+    HalTftDisplay
+lib_deps_external =
+    ;bodmer/TFT_eSPI @ ~2.4.70
+    https://github.com/nhjschulz/TFT_eSPI

```

Comment:
Is the bodmer/TFT_eSPI not working for you?

Reply:
It is, it sometimes did strange things when compiling - went back to bodmer